src/components/resumption/src/last_state_impl.cc

```diff
@@ -62,13 +62,17 @@ void LastState::SaveToFileSystem() {
   DCHECK(file_system::Write(app_info_storage_, char_vector_pdata));
 }
 
-void LastState::LoadFromFileSystem() {
+Json::Value& LastStateImpl::GetDictionary() {
+  return dictionary_;
+}
+
+void LastStateImpl::LoadFromFileSystem() {
```

Comment:
@dcherniev Propose to change the name `LoadFromFileSystem` -> `LoadStateFromFileSystem`